Objects/descrobject.c

```diff
@@ -1967,6 +1967,11 @@ static PyObject *
 ga_getitem(PyObject *self, PyObject *item)
 {
     gaobject *alias = (gaobject *)self;
+    // do a lookup for __parameters__ so it gets populated
```

Comment:
Maybe skip this if `alias->parameters` is already populated? The machinery for `PyObject_GetAttrString` is pretty heavy.